src/core/service.c

```diff
@@ -1584,7 +1532,7 @@ static int service_spawn(
         if (r < 0)
                 return r;
 
-        our_env = new0(char*, 10);
+        our_env = new0(char*, 11);
```

Comment:
this should be 12 if i count correctly, including the trailing NULL sentinel

Reply:
yep, forgot about the NULL